CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Add update-index-settings allowlist for searchable snapshot ([#5907](https://github.com/opensearch-project/OpenSearch/pull/5907))
 - Replace latches with CompletableFutures for extensions ([#5646](https://github.com/opensearch-project/OpenSearch/pull/5646))
 - Add support to disallow search request with preference parameter with strict weighted shard routing([#5874](https://github.com/opensearch-project/OpenSearch/pull/5874))
+- Added support to apply index create block ([#4603](https://github.com/opensearch-project/OpenSearch/issues/4603))
```

Comment:
This entry no longer exists in the changelog on main. I think it got lost because it was committed to main prior to the release of 2.5 but then not backported. Can you add it back to the main branch? (It's good to keep these sections in sync across branches to know what is where and to avoid conflicts on cherry-picks)